GVRf/Framework/framework/src/main/java/org/gearvrf/GVRSceneObject.java

```diff
@@ -1219,6 +1219,13 @@ public final BoundingVolume expandBoundingVolume(
     public final BoundingVolume expandBoundingVolume(final Vector3f center, final float radius) {
         return expandBoundingVolume(center.x, center.y, center.z, radius);
     }
+    
+    /*
+     * Check if the object is culled in the frame or not
```

Comment:
What good is this method to be exposed to end users? If that was to be exposed it probably would take an event-based mechanism.